test/branches_v1/meta.test.ts

```diff
@@ -0,0 +1,70 @@
+import * as dotenv from 'dotenv'
+import axios from 'axios'
+import MockAdapter from 'axios-mock-adapter'
+import { v1 } from '../../src/tillhub-js'
+import { initThInstance } from '../util'
+import faker from 'faker'
+import { BranchesMetaFailed } from '../../src/v1/branches'
+dotenv.config()
+
+const legacyId = '4564'
+
+const mock = new MockAdapter(axios)
+
+beforeEach(() => {
+  if (process.env.SYSTEM_TEST !== 'true') {
+    mock.onPost('https://api.tillhub.com/api/v0/users/login').reply(() => {
+      return [200, { token: '', user: { id: '123', legacy_id: legacyId } }]
+    })
+  }
+})
+
+afterEach(() => {
+  mock.reset()
+})
+
+const correlationId = faker.datatype.uuid()
+
+describe('v1: Branches', () => {
+  it('meta', async () => {
+    if (process.env.SYSTEM_TEST !== 'true') {
+      mock
+        .onGet(`https://api.tillhub.com/api/v1/branches/${legacyId}/meta`)
+        .reply(() => {
+          return [
+            200,
+            {
+              count: 1,
+              results: [{ correlationId }]
```

Comment:
The results are not realistic. Here's a good example for meta test:
https://github.com/tillhub/tillhub-sdk-javascript/blob/master/test/storefronts/available-products-meta.test.ts